pkg/specgenutil/specgen.go

```diff
@@ -592,10 +592,10 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *entities.ContainerCreateOptions
 		s.DependencyContainers = c.Requires
 	}
 
-	// TODO
-	// outside of specgen and oci though
-	// defaults to true, check spec/storage
-	// s.readonly = c.ReadOnlyTmpFS
+	// Only add ReadWrite tmpfs mounts iff the container is ReadOnly and
+	// the user did not disable the --read-only-tmpfs flag.
```

Comment:
This comment is based on the assumption that `ReadWriteTmpFS'`s default is `true`. But, the default value is defined somewhere else (in `cmd`). If for some reason, the default is changed, no one will know to change this comment and it will left with wrong information. Maybe it's better to leave the `user` out of this commnet

Reply:
I want future maintainers to understand what this logic is doing, so I attempted to improve the comment.